test/cartesian/CartesianGrid.spec.tsx

```diff
@@ -430,34 +587,27 @@ describe('<CartesianGrid />', () => {
             scale: scaleLinear(),
           };
           render(
-            <Surface width={500} height={500}>
+            <AreaChart width={300} height={200} margin={chartMargin}>
               <CartesianGrid
                 x={0}
                 y={0}
                 width={500}
                 height={500}
                 yAxis={yAxis}
                 horizontalCoordinatesGenerator={horizontalCoordinatesGenerator}
-                chartWidth={300}
-                chartHeight={200}
                 offset={offset}
                 syncWithTicks={syncWithTicks}
               />
-            </Surface>,
+            </AreaChart>,
           );
 
-          const yAxisExpected = {
-            ...yAxis,
-            ticks: undefined,
-          };
-
           expect(horizontalCoordinatesGenerator).toHaveBeenCalledOnce();
           expect(horizontalCoordinatesGenerator).toHaveBeenCalledWith(
             {
-              yAxis: yAxisExpected,
+              yAxis: undefined,
               width: 300,
               height: 200,
-              offset,
+              offset: { bottom: 11, brushBottom: 11, height: 175, left: 12, right: 13, top: 14, width: 275 },
```

Comment:
We have extract a constant in the top `expectedOffset`. Should we also use it here?

Reply:
Done